go/service/chat_local.go

```diff
@@ -1543,3 +1565,9 @@ func (h *chatLocalHandler) FindConversationsLocal(ctx context.Context,
 	res.RateLimits = utils.AggRateLimits(res.RateLimits)
 	return res, nil
 }
+
+// perfLog logs function call durations in ms.
+//    example usage: defer h.perfLog("MyFunc", time.Now())
+func (h *chatLocalHandler) perfLog(name string, start time.Time) {
+	h.G().Log.Debug("ChatPerf %s: %d", name, time.Since(start)/time.Millisecond)
```

Comment:
Yeah, that's similar to the Trace func we have, but sometimes we forget to call the function it returns :) ...i.e. compiler doesn't complain about `h.perfLog("mike")`.